server/src/pages/shotindex/view.js

```diff
@@ -68,6 +71,15 @@ class Body extends React.Component {
     );
   }
 
+  renderNoDeviceId() {
+    return (
+      <div className="large-icon-message-container" key="no-shots-found">
+        <div className="large-icon logo-no-shots" />
```

Comment:
`/creating` has one that's in CSS, so it'd probably be easy to use.  Could be followup Stretch bug.